test/authserver.jl

```diff
@@ -1,4 +1,5 @@
 using HTTP, Random, JSON
+import Pkg: TOML
```

Comment:
Some conflicts here with #26, so if one gets merged, the other should be updated before merging it.

Reply:
```suggestion
using TOML
```